Exiled.API/Features/Player.cs

```diff
@@ -359,6 +360,15 @@ public string CustomInfo
             set => ReferenceHub.nicknameSync.Network_customPlayerInfoString = value;
         }
 
+        /// <summary>
+        /// Gets or sets the range at which this player's info can be viewed by others.
+        /// </summary>
+        public float InfoViewRange
```

Comment:
Need testing to know if it's not only client side or if it's required to send a FakeMirror message

Reply:
It's networked so I'd imagine not but idk